widget/menu_item.go

```diff
@@ -392,33 +415,63 @@ func shortcutColor(th fyne.Theme) color.Color {
 	return color.NRGBA{R: uint8(r), G: uint8(g), B: uint8(b), A: uint8(a)}
 }
 
-func textsForShortcut(s fyne.KeyboardShortcut, th fyne.Theme) (texts []*canvas.Text) {
-	b := strings.Builder{}
+// textsForShortcut return texts for rendering a shortcut.
+func textsForShortcut(s fyne.KeyboardShortcut, th fyne.Theme) []*canvas.Text {
+	return textsForShortcutAndOS(s, th, runtime.GOOS)
+}
+
+// textsForShortcut return texts for rendering a shortcut for a specific OS.
+func textsForShortcutAndOS(s fyne.KeyboardShortcut, th fyne.Theme, os string) []*canvas.Text {
+	var b strings.Builder
```

Comment:
I would rather keep the previous `b := strings.Builder{}` syntax if you don't mind. That's more consistent with other code in the repo as far as I know.

Reply:
I chose this form, because it is unnecessary to initialize a string builder and I think this form is more Go idiomatic (e.g. this is what is used in the [official go documentation](https://pkg.go.dev/strings#example-Builder). And you actually find both variants in the repo (e.g. [here](https://github.com/fyne-io/fyne/blob/81b3cd1496f0a41dc8dee09f8f60722575274824/widget/markdown.go#L176).
So I would prefer to keep this change, would that be ok with you?

Reply:
There is no code difference between the two as all Go variables, unlike in C, are initialized to matter what so there is no performance win here. I much prefer using `:=` where possible as there is less words to parse until I get to the type.

Reply:
I am aware that both variants are equivalent and I understand that you personally prefer the other form. My point is that using var form is more idiomatic Go code. e.g. the Go standard library uses the var form [exclusively](https://github.com/search?q=repo%3Agolang%2Fgo+%2Fvar+%5Cw%2B+strings.Builder%2F&type=code).

Reply:
I see your point and I appreciate that the compiler does aeon thing else but let's keep consistent with our own code base please. https://github.com/search?q=repo%3Afyne-io%2Ffyne%20strings.Builder&type=code